tests/unit/models/cart-model-test.js

```diff
@@ -96,6 +96,29 @@ test('it creates a line item when you add a variant', function (assert) {
   });
 });
 
+test('it creates a line item when you add a variant', function (assert) {
+  assert.expect(3);
+
+  const done = assert.async();
+
+  const quantity = 2;
+
+  const variant = singleProductFixture.product_listing.variants[1];
+
```

Comment:
Oh, I wasn't suggesting to test if it exists. I just meant that we should assert the transition, so test that our assumptions about initial state are valid, so that we can ensure that a change has occurred